### ISSUE TITLE
5.1 upgrade failed?

### ISSUE THREAD

**Issue by xme** [ail-project/ail-framework#171]
I just wanted to upgrade my AIL instance to 5.1.
When performing a « LAUNCH -u », I got this:
```
Updating 4567c9d4..b4d536f1
Fast-forward
 bin/importer/PystemonImporter.py                    |  9 +++++++--
 bin/importer/abstract_importer.py                   |  2 +-
 bin/lib/Investigations.py                           | 21 +++++++++++++++++----
 bin/lib/crawlers.py                                 |  3 ++-
 bin/lib/objects/Items.py                            |  7 +++----
 bin/lib/objects/Screenshots.py                      |  2 +-
 bin/lib/objects/abstract_daterange_object.py        |  7 ++++---
 bin/lib/objects/abstract_object.py                  |  2 +-
 var/www/blueprints/investigations_b.py              | 15 ++++++++++++---
 .../investigations/view_investigation.html          | 18 ++++++++++++------
 .../modals/investigations_register_obj.html         | 16 ++++++++++++----
 11 files changed, 72 insertions(+), 30 deletions(-)


------------------------------------------------------------------
-                 Launching Update: v5.1                         -
--  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
Needs the env var AIL_ARDB. Run the script from the virtual environment.

------------------------------------------------------------------
                   Update Error: v5.1
------------------------------------------------------------------
Rolling back to safe commit: 175963e371509435f2568454ce8e8611a3a3dbf2
b''
```
But ARDB is gone or I missed something?

**Comment by Terrtia** [comment]
Hi @xme !
Thanks for the report !
Fixed with fce63d52a0a20170943d68b4d8de9ec957747c5e

**Comment by xme** [comment]
Tx! Upgraded. Now, it does not complain about the missing variable but it seems that AIL entered an infinite loop to start the ARDB instance!? But it's gone, right?

```
	* Launching ARDB servers
	6382 ARDB not ready
	6382 ARDB not ready
	6382 ARDB not ready
	6382 ARDB not ready
	6382 ARDB not ready
	6382 ARDB not ready
	6382 ARDB not ready
	6382 ARDB not ready
	6382 ARDB not ready
	6382 ARDB not ready
	6382 ARDB not ready
	6382 ARDB not ready
	6382 ARDB not ready
	6382 ARDB not ready
```

**Comment by Terrtia** [comment]
Indeed, I missed this one 
Fixed with 0fa0984ec4ffa3ed19383aef1dd071a8a1e5313b

**Comment by xme** [comment]
Much better! Tx! :-)
